### ISSUE TITLE
Use of triggers instead of buttons

### ISSUE THREAD

**Issue by OroArmor** [6391-Ursuline-Bearbotics/2020_UARobotics_Infinite_Recharge#8]
https://github.com/6391-Ursuline-Bearbotics/2020_UARobotics_Infinite_Recharge/blob/09c9c3baa37dc89dd5b4661b4605c208c920fbed/src/main/java/frc/robot/RobotContainer.java#L103-L105

**Comment by jasondaming** [comment]
I completely agree that as written these should be Triggers because they are use as triggers in the code.  If however I wanted to use some of the features of buttons that aren't available to triggers you wouldn't have a fundamental opposition to making these buttons would you?

This was discussed here:
https://www.chiefdelphi.com/t/digtial-input-firing-a-command/378369

**Comment by OroArmor** [comment]
Ah ok I understand now, i believe that there was a way to do it, but I don't remember completely. For this context that would work perfectly fine.
